etc/conf.templates/standalone/10-fdb.conf.template

```diff
@@ -0,0 +1,62 @@
+//
+//   Copyright 2023  SenX S.A.S.
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+
+
+/////////////////////////////////////////////////////////////////////////////////////////
+//
+// F o u n d a t i o n D B
+//
+/////////////////////////////////////////////////////////////////////////////////////////
+
+
+##
+## This the API version to use for FoundationDB
+##
+fdb.api.version = 710
```

Comment:
The requirement is 7.2 since we need tenants. Results are undefined if using 6.x or any prior version. The actual api version will depend on the FDB version deployed so it will need to be changed in some cases.

We consider the users sufficiently competent if they consider using FDB, so no need to treat them as children and hold their hand all the time.